### DISCUSSION TITLE
function mssql_fdw_handler() does not exist

### DISCUSSION THREAD

**Discussion by deluxebear** [supabase/supabase#29406] [Questions]
use supabase wrappers

https://fdw.dev/catalog/mssql/

create foreign data wrapper mssql_wrapper
  handler mssql_fdw_handler
  validator mssql_fdw_validator

> ERROR:  function mssql_fdw_handler() does not exist

**Comment by silentworks** [comment]
You need to first enable fdw on this page https://supabase.com/dashboard/project/_/database/wrappers and then enable the mssql on by filling in the settings.
